### DISCUSSION TITLE
Announcements

### DISCUSSION THREAD

**Discussion by sh1zicus** [sh1zicus/dots-hyprland-old#37] [Announcements]
Updates from maintainers

**Comment by sh1zicus** [comment]
![image](https://github.com/user-attachments/assets/99938f6b-bf18-4efd-91a4-f8835e9d84a0)
GJS Youtube downloader 🧐

**Reply by PharmaRacist** [reply]
love it 🖤

**Comment by sh1zicus** [comment]
![Вставленное изображение](https://github.com/user-attachments/assets/4ec84d2e-f390-4d36-a5ca-d6d397101b4b)
Simple timer

**Comment by PharmaRacist** [comment]
why dont you implement those in ags ?would be great to add one with the ModuleCalendar

**Reply by sh1zicus** [reply]
> why dont you implement those in ags ?would be great to add one with the ModuleCalendar

I want to do both, and I think I should keep some big things separate from ags

**Reply by PharmaRacist** [reply]
can you make this espicially inside ags too 
overall i think the right sidebar need some work like the network and bluetooth are all laggy and inconvenient the sound mixer is useless
